src/components/article-list.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+import getArticles from "../api";
+
+function ArticleList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((data) => {
+      setArticles(data);
+    });
+  }, []);
```

Comment:
Would like to see use of a loading state while im waiting for this data to be fetched. Let your user know that the data is coming, otherwise they are just left wondering whether anything is happening